src/views/About.vue

```diff
@@ -72,16 +69,47 @@
 </template>
 <script>
 import AmbListItem from '@/components/shared/ListItem.vue'
+import { PEER_CONNECTED } from '@/store/mutation-types'
 import { mapState } from 'vuex'
+import { FETCH_EDGE_DEVICE_DETAILS } from '../store/action-types'
+import { EdgeAPI } from '../remote/edgeAPI'
 
 export default {
   components: {
     AmbListItem
   },
   computed: {
     ...mapState({
-      version: state => state.version
+      version: (state) => state.edgeDevice.version,
```

Comment:
@vickywane this replaces the PWA version display with edge device version display. Not a good UX design.
The About page should continue to show the PWA version as before. Maybe we should add a clarification that this is the UI app version so its clear. Edge device version should be shown only next to a device details section, which is currently in the Settings.vue page.